__tests__/app.test.js

```diff
@@ -42,3 +42,32 @@ describe('/api/topics', () => {
   })
 });
 
+describe('/api/articles/:article_id', () => {
+  test('GET:200 gets a single article by article_id when the ID exists', () => {
+    return request(app)
+      .get('/api/articles/2')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article_id).toBe(2);
```

Comment:
we do also expect the article to have other keys so would be great to check that they also exist but well done for explicitly checking the id is the one requested!